### ISSUE TITLE
Table of contents subsections numbering reversed

### ISSUE THREAD

**Issue by Lancer555** [eyalroz/technion-iit-thesis#28]
Hi,
All numbering, except for the section numbering, appears in reverse order.
For example, the table of contents appears like this:
1       Introduction
2       Background
         1.2    Some subsection 1
         2.2    Some subsection 2
                  1.2.2    Some subsubsection 1

Instead of like this:
1       Introduction
2       Background
         2.1    Some subsection 1
         2.2    Some subsection 2
                  2.2.1    Some subsubsection 1

Would very appreciate your support!

**Comment by eyalroz** [comment]
I can't reproduce this on my machine (Devuan ASCII 2.0 ~= Debian Stretch), which uses TeXLive version 2016.20170123-5, on an amd64 machine.

Please try providing information regarding your platform and exact reproduction instructions.

**Comment by Lancer555** [comment]
Hi,
I'm using the Thechnion template in overleaf:
https://www.overleaf.com/latex/templates/technion-iit-thesis-template/rzqpztzgppwz
you can either "Download PDF" and already see it, or you can "Open as
Template" and see it there as well

**Comment by eyalroz** [comment]
I know Overleaf has some directionality issues with their PDF generation.

Do you get this bug when you use the template directly? That is, download a zip from here and build the PDF on your system?

**Comment by Lancer555** [comment]
You mean, download zip from github and use it in overleaf?

**Comment by eyalroz** [comment]
No, I mean download the zip yourself and use it on your own system. However, I'll also look into figuring out what's wrong on Overleaf...

**Comment by Lancer555** [comment]
It seams that when choosing LuaLaTex as the compiler, instead of  pdfLaTex,
the problem go away.

**Comment by eyalroz** [comment]
@Lancer555 : The compiler should be xelatex...

**Comment by yotamitai** [comment]
Experiencing this as well with the overleaf template when compiling with overleaf platform with xelatex compiler. (LuaLaTex solves this but creates some other issues).
Haven't tried reproducing on local machine. wonder if there is a way to fix this in overleaf.

**Comment by Zendelo** [comment]
On local (linux) machine the bug doesn't exists (unless one have some local issues)
In Windows you'll have other issues, some can be solved by changing the font to David (from David CLM), beside that it works well (on Windows as well)
The bug seems to be only in Overleaf.
It seems that all subsections are in wrong direction (not only in the table of contents) 

It seems to me that a possible cause can be package versions difference, on my linux machine I have:

From the log file:
```
This is XeTeX, Version 3.14159265-2.6-0.99998 (TeX Live 2017/Debian) (preloaded format=xelatex 2020.2.29)
...
Package: polyglossia 2015/03/25 v1.42.4 Alternative to Babel for XeLaTeX and Lu
aLaTeX
```


On Overleaf according to the logs they use:
```
This is XeTeX, Version 3.14159265-2.6-0.99996 (TeX Live 2016/Debian) (preloaded format=xelatex 2018.1.2)
...
Package: polyglossia 2015/03/25 v1.42.4 
```

**Comment by eyalroz** [comment]
@Zendelo : That could be it, I suppose.

Anyway, please open a separate issue about David vs David CLM.
